### PR TITLE
Adds missing dependency

### PR DESCRIPTION
**Pull request by arcanis**
Required here: https://github.com/lingui/js-lingui/blob/next/packages/macro/src/index.ts#L2

### CODE REVIEW

File: packages/macro/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
   ],
   "dependencies": {
     "@babel/runtime": "^7.9.2",
+    "@lingui/conf": "^3.0.0-11",
```
Comment (tricoder42) [inline packages/macro/package.json:32]: Good catch! Just use placeholders for `@lingui/*` dependency versions, which is replaced with the correct version during build:

```suggestion
    "@lingui/conf": "0.0.0-managed-by-release-script",
```

**Comment by tricoder42** [comment]
Thanks a lot 👍

**Comment by arcanis** [comment]
Btw, note that Yarn 2 now [automatically the dependency ranges](https://yarnpkg.com/features/release-workflow#auto-updated-dependencies) if needed 🙂

**Comment by tricoder42** [comment]
Is it released already? I still use yarn `1.22.4` but I haven't checked for a while

**Comment by arcanis** [comment]
Yep it got released in January - the latest on npm is still 1.22.4 because we switched to a per-project package manager versioning. More details here: https://yarnpkg.com/getting-started/install

**Comment by tricoder42** [comment]
Nice! Thanks a lot! 🙏

**Pull request merged**